package.json

```diff
@@ -74,7 +74,7 @@
     "jest": "^24.8.0",
     "jest-junit": "^7.0.0",
     "mockdate": "^2.0.5",
-    "node-sass": "^4.12.0",
+    "node-sass": "^4.13.1",
```

Comment:
Don't update `node-sass`. If you look at [their support chart](https://github.com/sass/node-sass#node-sass), the minor version (13), means the node version `node-sass` is compatible with.